### PR TITLE
[COR-188] Golang SDK docs

### PR DESCRIPTION
**Pull request by rukolahasser**
A detailed guide to building a connector.

### CODE REVIEW

File: examples/compound/README.md [hunk 1]
```diff
@@ -1,2 +1,288 @@
...
+3. protoc
+4. protoc-gen-go
+5. abigen
+6. solc
+
+## Overview
+
+We will be building a source connector. As a reminder,a source connector extracts/ingests, cleans, and generalizes data
+from an external source, usually a blockchain, smart contract, or API. It then passes on this data to the Nakji message
+queue. **It is highly recommended to use the example compound connector as the template to build your own connector.**
```
Comment (cnkarz) [inline examples/compound/README.md:20]: >compound connector

can you rephrase it to something like `connector built for Compound lending platform`?

File: examples/compound/README.md [hunk 2]
```diff
@@ -1,2 +1,288 @@
...
+
+Next, spin up kafka by running:
+
+```shell
+docker-compose up -d
+```
+
+Now start the connector locally by running the `main.go` file. (If you are running to “connection refused” error, make
+sure docker is running)
+
```
Comment (cnkarz) [inline examples/compound/README.md:267]: I think they also need to run `protoregistry`

**Review by cnkarz — APPROVED** [review]
Left 2 remarks

File: examples/compound/README.md (hunk 2 above)
Comment (rukolahasser) [reply examples/compound/README.md:267]: No they don't, as long as the `env` is set to `dev`

File: examples/compound/README.md (hunk 1 above)
Comment (rukolahasser) [reply examples/compound/README.md:20]: Will refine the whole docs later.
